addons/hr_homeworking_calendar/static/src/calendar/common/calendar_common_renderer.scss

```diff
@@ -7,6 +7,15 @@
     }
 }
 
+.o_worklocation_span {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    text-wrap: nowrap;
+    max-width: 4rem;
+    margin-right: 1rem;
+}
```

Comment:
So in practice we already have selectors for a lot of effects. For instance here: `overflow-hidden` `text-nowrap` `me-3` ... or even `text-truncate` that covers three lines

One very useful file is `bootstrap.css`, you can search in there your desired selector and you will find a lot of useful things :)

This way, we keep most selectors in the templates and keep scss under control :)